### PR TITLE
Publish timere-parse.0.0.1

### PR DESCRIPTION
**Pull request by darrenldl**
Timere natural language parsing component

Homepage: https://github.com/daypack-dev/timere
Repo: git+https://gitlab.com/daypack-dev/timere.git
Issue tracker: https://github.com/daypack-dev/timere/issues

### CODE REVIEW

File: packages/timere-parse/timere-parse.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Timere natural language parsing component"
+description: """
```
Comment (avsm) [inline packages/timere-parse/timere-parse.0.0.1/opam:3]: If you have time, even a brief description would be appreciated. It's important to provide hints to other users of opam as they browse

Comment (darrenldl) [reply packages/timere-parse/timere-parse.0.0.1/opam:3]: Ah yep. I've added a short one for now.

Comment (avsm) [reply packages/timere-parse/timere-parse.0.0.1/opam:3]: thanks!

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**